Client/src/Pages/Settings/index.jsx

```diff
@@ -37,10 +37,30 @@ const Settings = ({ isAdmin }) => {
   const [form, setForm] = useState({
     ttl: checkTTL ? (checkTTL / SECONDS_PER_DAY).toString() : 0,
   });
+  const [version,setVersion]=useState("");
   const [errors, setErrors] = useState({});
   const dispatch = useDispatch();
   const navigate = useNavigate();
 
+  //Fetching latest release version from github
+  useEffect(() => {
+		const fetchLatestVersion = async () => {
+			try {
+				const response = await fetch(
+					"https://api.github.com/repos/bluewave-labs/bluewave-uptime/releases/latest"
+				);
```

Comment:
We have a `NetworkService` class in which we use `Axios` to make all of our network requests.  Can you refactor this to make use of the `NetworkService` so that all of our network requests are in one place?